cpp/test/test.cpp

```diff
@@ -42,6 +48,30 @@ std::string vec_to_string(RustVeci8 a)
     return std::string(reinterpret_cast<char const *>(&a[0]), a.size());
 }
 
+std::string random_string(std::size_t length)
```

Comment:
Just curious, did you figure out if this was actually random?

Reply:
If you look at the commit history you'll see that this is a different method (which is actually random). So in short, yes this is random.